tests/conftest.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from app import create_app, db
+from flask.signals import request_finished
+from app.models.planet import Planet
+
+
+@pytest.fixture
+def app():
+    app = create_app({"TESTING": True})
+
+    @request_finished.connect_via(app)
+    def expire_session(sender, response, **extra):
+        db.session.remove()
+
+    with app.app_context():
+        db.create_all()
+        yield app
+
+    with app.app_context():
+        db.drop_all()
+
+
+@pytest.fixture
+def client(app):
+    return app.test_client()
+
+
+@pytest.fixture
+def two_planets(app):
+    Jupiter = Planet(
+        name="Jupiter",
+        description="Jupiter has sixty-seven moons",
+        distance_from_sun=48.4
+    )
+    Mercury = Planet(
+        name="Mercury",
+        description="Mercury has zero moons",
+        distance_from_sun=3.5
+    )
+    db.session.add(Jupiter)
+    db.session.add(Mercury)
+    db.session.commit()
```

Comment:
Something you could do here is to create a list and add the 2 planets to it, then return the list. That way when a unit test brings in this fixture two_planets you actually get the list of 2 saved planets. Right now this fixture doesn't return anything.

So that would look like

list_of_planets = [jupiter, mercury]